src/components/BannerHighlight/BannerHighlight.vue

```diff
@@ -54,69 +54,77 @@ export default {
 
 <style lang="scss">
 .banner-highlight {
+    color: var(--color-company);
     display: block;
     position: relative;
     height: 80vh;
+}
 
-    .block-text {
-        position: absolute;
-        top: 0;
-        left: 0;
-        height: 100%;
-        width: 100%;
-        z-index: 10;
-
-        padding: 50px;
-        box-sizing: border-box;
-
-        display: flex;
-        flex-direction: column;
-        justify-content: flex-end;
-    }
-    .text {
+.block-text {
+    position: absolute;
+    top: 0;
+    left: 0;
+    height: 100%;
+    width: 100%;
+    z-index: 10;
+
+    padding: 110px 50px;
+    box-sizing: border-box;
+
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+}
+.text {
+    margin: 0;
+    font-size: 89px;
+    padding-bottom: 43px;
+    font-weight: 300;
+}
+
+.prompt-container {
+    position: absolute;
+
+    .prompt {
+        font-size: 28px;
+        overflow: hidden;
+        transition: opacity 200ms ease;
+        opacity: 0;
+        padding: 0 1rem 0 0;
         margin: 0;
-        font-size: 89px;
-        padding-bottom: 50px;
-        font-weight: 300;
+        padding-left: 8px;
     }
-    
-    .prompt-container {
+
+    .arrow {
         position: absolute;
-        
-        .prompt {
-            font-size: 28px;
-            overflow: hidden;
-            transition: opacity 200ms ease;
-            opacity: 0;
-            padding: 0 1rem 0 0;
-        }
+        left: 8px;
+        top: 50%;
 
-        .arrow {
-            position: absolute;
-            left: 0;
-            top: 50%;
+        transform: translateY(-50%);
+        transition: left 200ms ease;
+    }
 
-            transform: translateY(-50%);
-            transition: left 200ms ease;
-        }
+    &:hover .prompt {
+        opacity: 1;
+    }
 
-        &:hover .prompt {
-            opacity: 1;
-        }
+    &:hover .arrow {
+        left: 100%;
```

Comment:
This should be inside a $has-hover media query, also animate the translate, not left. In general, animating transforms is way smoother than a left/right/top etc.